docs/ruleset/core-ruleset.md

```diff
@@ -0,0 +1,116 @@
+# AsyncAPI Core Ruleset
+
+The core ruleset, which validates the overall structure of the specification.
+
+> **Note**
+> These rules will apply to each version, starting with `2.0.0`.
+
+## Rules
+
+### asyncapi-is-asyncapi
+
+The input must be a document with a supported version of AsyncAPI.
+
+#### Good example
```

Comment:
Nit: Not sure if @alequetzalli already mentioned it, but I don't think the terms `Good` or `Bad` are correct here. Maybe a `Valid/Invalid example` or similar will look more clear and more precise.